authz/sdk_end2end_test.go

```diff
@@ -422,20 +446,8 @@ func (s) TestSDKFileWatcher_ValidPolicyRefresh(t *testing.T) {
 	}
 
 	// Verifying authorization decision.
-	numRetries := 0
-	reloadSuccess := false
-	var gotStatus *status.Status
-	for numRetries <= 20 {
-		_, err = client.UnaryCall(ctx, &pb.SimpleRequest{})
-		if gotStatus = status.Convert(err); gotStatus.Code() == valid2.wantStatusCode && gotStatus.Message() == valid2.wantErr {
-			reloadSuccess = true
-			break
-		}
-		time.Sleep(100 * time.Millisecond)
-		numRetries++
-	}
-	if reloadSuccess == false {
-		t.Fatalf("error want:{%v %v} got:{%v %v}", valid2.wantStatusCode, valid2.wantErr, gotStatus.Code(), gotStatus.Message())
+	if got := verifyValidReload(ctx, client, valid2.wantStatusCode, valid2.wantErr); got != nil {
+		t.Fatalf("error want:{%v %v} got:{%v %v}", valid2.wantStatusCode, valid2.wantErr, got.Code(), got.Message())
```

Comment:
Just log the errors.  That's better (less verbose) than individually logging the code and message.

Reply:
Done.